### PR TITLE
[#178] Add missing .editorconfig

### LINKED ISSUE #178: No .editorconfig
This project does not have a `.editorconfig`. 

It helps maintain consistent coding styles for multiple developers working on the same project across various editors and IDEs.

More info at https://editorconfig.org/

### PR DESCRIPTION
**Pull request by waynee95**
Resolves #178

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+tab_width = 4
+
+[*.gradle]
```
Comment (dargmuesli) [inline .editorconfig]: Do we even have such a file? :0

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+indent_style = tab
+
+[*.java]
+indent_style = tab
```
Comment (dargmuesli) [inline .editorconfig]: Shouldn't this be spaces?

File: .editorconfig (hunk 1 above)
Comment (waynee95) [reply .editorconfig]: Whoops. This is my goto editorconfig for java and I have been only using gradle so far.. I will delete this.

File: .editorconfig (hunk 2 above)
Comment (waynee95) [reply .editorconfig]: Hmmm.. :thinking:

**Pull request merged**